### DISCUSSION TITLE
when is the next collector layer release?

### DISCUSSION THREAD

**Discussion by enuret** [open-telemetry/opentelemetry-lambda#1187] [Q&A]
Hi, we need a 0.94 version with this change: https://github.com/open-telemetry/opentelemetry-collector/pull/9411 
I wanted to ask when is the next release expected?

**Comment by tylerbenson** [comment]
Latest is available now:
https://github.com/open-telemetry/opentelemetry-lambda/releases/tag/layer-collector%2F0.6.0
